llm_agents/searx.py

```diff
@@ -0,0 +1,88 @@
+import os
+from typing import Any
+from llm_agents.tools.base import ToolInterface
+import requests
+from urllib.parse import urlencode
```

Comment:
Seems unused

Reply:
Correct, this was an artifact from testing with POST. Resolved in latest commit